site/content/admin-guide/common-config.md

```diff
@@ -0,0 +1,31 @@
++++
+title = "Common Configuration"
+weight = 300
++++
+
+Most Hono components support a common set of configuration options. This section
+describes those options. <!--more-->
+
+Each component which supports the following options explicitly states so. If
+it doesn't, then these options are not supported by this component.
+
+## Using epoll() on Linux 
+
+Using `epoll()` on Linux may provide better performance for application which
+have a high I/O throughput. Especially when the application supports an
+asynchronous I/O model. This is true for most Hono applications.
+
+Netty allows the use of `epoll()` on Linux x86_64 based systems. Hono provides
+a build profile for enabling this support, and allows to configure Vert.x to
+make use of this.
+
+Only when the Maven profile was active during the build **and** the Vert.x
```

Comment:
why do we need to activate the profile? The CQ should allow us to include the libs by default, shouldn't it? Using the property, the user can still decide if he wants to enable it ...

Reply:
Yes, we could enable this for the docker images explicitly. However we don't know if the user builds this on 64bit linux. So, when releasing the images, yes we can. By default, I wouldn't do that.